README.md

```diff
@@ -18,7 +18,7 @@ rabbitmq-server
 ```bash
 composer create-project pborreli/composer-service -sdev
 cd composer-service
-php -S localhost:9090 -t web &
+php -S 0.0.0.0:9090 -t web &
```

Comment:
This will make the instance also available on your internet ip - which is not what you want in most cases when doing development

Reply:
Yes, i'm totally agree with you. but with localhost we have to install php locally and not on a virtual machine or on docker containers

Reply:
If I use a virtual machine, it will not work for me